server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java

```diff
@@ -80,10 +80,10 @@
 
 public class GarbageCollectWriteAheadLogs {
   private static final Logger log = LoggerFactory.getLogger(GarbageCollectWriteAheadLogs.class);
+  private static final Map<HostAndPort,Long> firstSeenDead = new HashMap<HostAndPort,Long>();
 
   private final AccumuloServerContext context;
   private final VolumeManager fs;
-  private final Map<HostAndPort,Long> firstSeenDead = new HashMap<HostAndPort,Long>();
```

Comment:
Yeah, I was looking at the test cases to see about implementing something to expose the bug and ensure it gets fixed, but I am at a loss (I've spent WAY more time using Accumulo than digging through the source code).

I tested it locally by:

- Starting single-instance Accumulo
- Starting another TServer
- Inserting data
- Killing the other tserver
- Watch the GC add it to the map
- Watch the next GC cycle delete the WAL file (I configured it to have a short delete time)
- Started the other tserver and add more data
- Kill the other tserver and watch the GC add it to the map on the first cycle, then delete it next cycle